react/src/components/relations-widget/Application.jsx

```diff
@@ -100,8 +100,8 @@ class Application extends React.Component {
         relation.entity1,
         relation.relation,
         relation.entity2,
-        relation.source.text,
-        relation.source.url,
+        relation.sources[0].text,
```

Comment:
As mentioned in the commit message, only exporting the first source from the list in this global export. If there's interest in exporting all sources for a given relation, we might add that download button inside the sources popup to export them separately.